src/Altinn.Platform/Altinn.Platform.Events/Events/Migration/v0.03/03-setup-grants.sql

```diff
@@ -0,0 +1,5 @@
+GRANT  USAGE  ON SCHEMA events TO platform_events;
+GRANT SELECT,INSERT,UPDATE,REFERENCES,DELETE,TRUNCATE,REFERENCES,TRIGGER ON ALL TABLES IN SCHEMA events TO platform_events;
+GRANT ALL ON ALL SEQUENCES IN SCHEMA events TO platform_events;
```

Comment:
is this really necessary? This grant should already exist in the schema, and I guess also cover all resources added after the grant was created.

Reply:
@acn-sbuad according to @tba76 it is. And locally i had to run it after I added new table.

Reply:
Was the table added through the scripts? If so this seems like a weak point in this solution.. shouldn' be nessecary to repeat this.

Reply:
@acn-sbuad yes. I have not retried. Not able to insert data in to the subscription table without this